utils/ankiexport.py

```diff
@@ -80,20 +88,92 @@ def export_card(self, media_file, audio_track, text_primary, text_secondary, tim
             return -2
 
         return 0
-        
 
-
-    def make_audio(self, media_file, audio_track, start, end, out_path):
+    def ffmpeg_audio(self, media_file, audio_track, start, end, out_path):
+        args = [
+                'ffmpeg',
+                '-y', '-loglevel', 'error',
+                '-ss', str(start),
+                '-to', str(end),
+                '-i', media_file,
+                '-acodec', 'mp3',
+                out_path
+                ]
+
+        error = None
+        proc = subprocess.Popen(args, cwd=self.mpv_cwd)
+        proc.wait()
+
+        # Check that image was saved
+        if not os.path.exists(out_path):
+            error = Errors.FFMPEG_AUDIO_ERROR
+        return error, proc
+
+    def mpv_audio(self, media_file, audio_track, start, end, out_path):
         args = [self.mpv_executable, '--load-scripts=no',                                       # start mpv without scripts
                 media_file, '--loop-file=no', '--video=no', '--no-ocopy-metadata', '--no-sub',  # just play audio
                 '--aid=' + str(audio_track),
                 '--start=' + str(start), '--end=' + str(end),
                 '--o=' + out_path]
 
-        return subprocess.Popen(args, cwd=self.mpv_cwd)
+        error = None
+        proc = subprocess.Popen(args, cwd=self.mpv_cwd)
+        proc.wait()
 
+        # Check that image was saved
+        if not os.path.exists(out_path):
+            error = Errors.FFMPEG_SCREENSHOT_ERROR
+        return error, proc
 
-    def make_snapshot(self, media_file, start, end, out_path):
+    def make_audio(self, media_file, audio_track, start, end, out_path):
+        # Default to using ffmpeg for audio
+        error, proc = self.ffmpeg_audio(media_file, audio_track, start, end, out_path)
+        # Fall back to mpv if ffmpeg fails
+        if error == Errors.FFMPEG_AUDIO_ERROR:
+            error, proc = self.mpv_audio(media_file, audio_track, start, end, out_path)
+
+        if error:
+            error = Errors.AUDIO_ERROR
+
+        return error, proc
+
+    def ffmpeg_screenshot(self, media_file, start, end, out_path):
+        args = [
+                'ffmpeg',
```

Comment:
This should use the ffmpeg executable from https://github.com/migaku-official/migaku-mpv/blob/063fe424754af72350d2ad6935c5bd57d35a1d3d/migaku_mpv.py#L783